From 461278899bc7da8033d6c93e7497184d641b258c Mon Sep 17 00:00:00 2001
From: Chih-Wei Huang <cwhuang@linux.org.tw>
Date: Fri, 1 Nov 2013 18:52:32 +0800
Subject: [PATCH 05/70] ACPICA: Add Android-IA string for _OSI method

This change adds a new _OSI string "Android-IA" for ASUS BIOS
to query if the OS supports Android features.
---
 drivers/acpi/acpica/utosi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpica/utosi.c b/drivers/acpi/acpica/utosi.c
index 902a47463abf99..a0bf8df011f542 100644
--- a/drivers/acpi/acpica/utosi.c
+++ b/drivers/acpi/acpica/utosi.c
@@ -72,6 +72,7 @@ static struct acpi_interface_info acpi_default_supported_interfaces[] = {
 	{"Windows 2017.2", NULL, 0, ACPI_OSI_WIN_10_RS3},	/* Windows 10 version 1709 - Added 02/2018 */
 	{"Windows 2018", NULL, 0, ACPI_OSI_WIN_10_RS4},	/* Windows 10 version 1803 - Added 11/2018 */
 	{"Windows 2018.2", NULL, 0, ACPI_OSI_WIN_10_RS5},	/* Windows 10 version 1809 - Added 11/2018 */
+	{"Android-IA", NULL, 0, ACPI_OSI_WIN_8},	/* Android-IA - Added 11/2013 */
 
 	/* Feature Group Strings */
 
-- 
2.17.1

